jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -126,7 +162,7 @@ public static boolean isCRIUSupportEnabled() {
 	 * 
 	 * @return return CRIUResult
 	 */
-	public static CRIUResult checkPointJVM(Path checkPointDir) {
+	public CRIUResult checkPointJVM(Path checkPointDir) {
 		CRIUResult ret = new CRIUResult(CRIUResultType.UNSUPPORTED_OPERATION, null);
 
 		if (null == checkPointDir) {
```

Comment:
over just throw an NPE stating that the `image directory` needs to be set with `setImageDir()`